unit-tests/source_tree_test.py

```diff
@@ -63,15 +67,15 @@ def as_list(self) -> List[str]:
                 return ['cp', str(self._filename), str(self.output[0])]
 
         db = SqliteStateDatabase(tmp_path)
-        emap: Dict[str, Union[Type[Task], Type[Command]]] = {
-            '.bar': DummyCommand
-        }
+        smap = PathMap([
+            (r'.*\.bar', DummyCommand),
```

Comment:
Wildcard when literal may have been intended.

Reply:
As above, I think this is the correct pattern to match any path to a file with the `.bar` extension